cli/js/dispatch.ts

```diff
@@ -66,6 +67,16 @@ export let OP_CWD: number;
 export let OP_FETCH_ASSET: number;
 export let OP_DIAL_TLS: number;
 export let OP_HOSTNAME: number;
+export let OP_OPEN_NATIVE_PLUGIN: number;
+
+const NATIVE_PLUGIN_ASYNC_HANDLER_MAP: Map<number, AsyncHandler> = new Map();
+
+export function setPluginAsyncHandler(
```

Comment:
This should ultimately be defined in `native_plugins.ts` but we can defer on that until we split ops into crates